buildtools/maven/maven.go

```diff
@@ -32,9 +37,82 @@ type Parent struct {
 }
 
 type Dependency struct {
-	Name    string
-	Version string
-	Failed  bool
+	Name string
+
+	GroupId    string `xml:"groupId"`
+	ArtifactId string `xml:"artifactId"`
+	Version    string `xml:"version"`
+
+	// Scope is where the dependency is used, such as "test" or "runtime".
+	Scope string `xml:"scope"`
+
+	Failed bool
+}
+
+type depsList []Dependency
+
+func (d depsList) toImports() []pkg.Import {
+	i := make([]pkg.Import, 0, len(d))
+	for _, dep := range d {
+		i = append(i, pkg.Import{
+			Resolved: pkg.ID{
+				Type:     pkg.Maven,
+				Name:     dep.Name,
+				Revision: dep.Version,
+			},
+		})
+	}
+	return i
+}
+
+// ToGraphDeps returns simply the list of dependencies listed within the manifest file.
+func (m *Manifest) ToGraphDeps() graph.Deps {
+	g := graph.Deps{
```

Comment:
[nit] We should avoid using one letter variable names. It forces us to find the initialization to understand what it is. This also goes for the method below.